src/analyzer/plugins/analyzerpluginsupport.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <QString>
+
+#include "analyzer/analyzer.h"
+#include "analyzer/plugins/analyzerplugin.h"
+
+struct AnalyzerPluginSupportInfo {
+    virtual QString matchAndSetPluginId(
```

Comment:
virtual is used if the child class implements the function. I think this is not the case here, so we should remove the virtual keyword.